frame/staking/src/tests.rs

```diff
@@ -3039,3 +3039,98 @@ fn set_history_depth_works() {
 		assert!(!<Staking as Store>::ErasTotalStake::contains_key(10 - 5));
 	});
 }
+
+/* Benchmarking Tests */
+
+use crate::benchmarking::{
+	create_validators_with_nominators_for_era,
+	create_validator_with_nominators,
+	create_nominator_with_validators,
+};
+
+#[test]
+fn create_validators_with_nominators_for_era_works() {
+	ExtBuilder::default().build().execute_with(|| {
+		let v = 10;
+		let n = 100;
+		let mut validator_iter = Validators::<Test>::enumerate();
+		let mut nominator_iter = Nominators::<Test>::enumerate();
+
+		// Clean up genesis validators and nominators to make test work
```

Comment:
I added a `has_stakers` to mock builder in offchain phragmen PR, maybe you can do something similar.

Reply:
yes, this is a good idea :)

Reply:
(I am going to hate myself when merging the conflicts later though :D)